.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
```

Comment:
we need `sudo: false` here , and then add `sudo: true` in the definition of the osx job